lib/metrics/StatsModel.js

```diff
@@ -88,14 +102,11 @@ class StatsModel extends StatsClient {
                 return cb(null, statsRes);
             }
 
-            if (requests.length) {
-                statsRes.requests = this._zip(requests).map(arr =>
-                    arr.reduce((acc, i) => acc + i, 0));
-            }
-            if (errors.length) {
-                statsRes['500s'] = this._zip(errors).map(arr =>
-                    arr.reduce((acc, i) => acc + i, 0));
-            }
+            statsRes.requests = this._zip(requests).map(arr =>
+                arr.reduce((acc, i) => acc + i));
```

Comment:
It does not hurt to set the initial value explicitly, you can keep the last `reduce()` argument.